### PR TITLE
Config Generation and README Updates

### PR DESCRIPTION
**Pull request by samuelattwood**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
 services:
-  helix:
-    container_name: helix
-    image: registry.helix-dev.synadia.io/helix:tag-release
-    hostname: helix
-    command: ["-c", "/conf/helix/helix.json", "server", "start"]
+  control-plane:
+    container_name: control-plane
+    image: registry.helix-dev.synadia.io/control-plane:tag-rc
```
Comment (caleblloyd) [inline docker-compose.yaml:6]: Prep for the release, will merge after tagged

```suggestion
    image: registry.helix-dev.synadia.io/control-plane:tag-release
```

File: demo/docker-compose.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 services:
-  helix:
-    container_name: helix
-    image: registry.helix-dev.synadia.io/helix:tag-release
-    hostname: helix
-    command: ["-c", "/conf/helix/rna.cue", "server", "start"]
+  control-plane:
+    container_name: control-plane
+    image: registry.helix-dev.synadia.io/control-plane:tag-rc
```
Comment (caleblloyd) [inline demo/docker-compose.yaml:6]: Prep for the release, will merge after tagged

```suggestion
    image: registry.helix-dev.synadia.io/control-plane:tag-release
```

File: generate-config.sh [hunk 3]
```diff
@@ -743,20 +1000,19 @@ if [[ "${HELM}" == "true" ]]; then
...
+  / __/ _ \| \| |_   _| _ \/ _ \| |    | _ \ |    /_\ | \| | __| |_ _| \| / __|_   _/_\ | |  | |   
+ | (_| (_) | .` | | | |   / (_) | |__  |  _/ |__ / _ \| .` | _|   | || .` \__ \ | |/ _ \| |__| |__ 
+  \___\___/|_|\_| |_| |_|_\\___/|____| |_| |____/_/ \_\_|\_|___| |___|_|\_|___/ |_/_/ \_\____|____|
 '
 
 echo "\
 helm repo add synadia https://connecteverything.github.io/helm-charts
 helm repo update
-helm upgrade --install helix -n helix --create-namespace\
+helm upgrade --install syn-cp -n syn-cp --create-namespace\
```
Comment (caleblloyd) [inline generate-config.sh:1012]:
```suggestion
helm upgrade --install control-plane -n syn-cp --create-namespace\
```

**Review by caleblloyd — APPROVED** [review]
Looks good, we can prep it for release!

File: generate-config.sh [hunk 4]
```diff
@@ -540,6 +635,112 @@ setup_registry_credentials() {
...
+
+    hostname=$(echo ${url} | awk -F[/:] '{print $4}')
+
+    response=$(prompt "Use Kubernetes Ingress?" "${regex_yn}" "true" "No")
+    if [[ "${response}" =~ ^[yY] ]]; then
+        ingress=$(add_kv_bool_to_object "enabled" true "${ingress}")
+        hostname=$(prompt "Ingress Hostname" "" "true" "${hostname}")
+        hostname="[\"${hostname}\"]"
+        ingress=$(add_json_to_object "hosts" "${hostname}" "${ingress}")
+        ingress_class=$(prompt "Ingress Class" "" "true" "nginx")
```
Comment (samuelattwood) [inline generate-config.sh:650]:
```suggestion
        ingress_class=$(prompt "Ingress Class (Optional)" "" "true")
        if [[ -n “${ingress_class}” ]]; then
            ingress=$(add_kv_to_object “className” “${ingress_class}” “${ingress}”)
        fi
```

Comment (samuelattwood) [reply generate-config.sh:650]: I wrote this on my phone - wanted to check it before committing

Comment (caleblloyd) [reply generate-config.sh:650]: we can put it in a follow-up, I think the quotes are smart quotes :laughing:

Comment (samuelattwood) [reply generate-config.sh]: Ha, just got home and fixed it

Comment (caleblloyd) [reply generate-config.sh]: :+1:

**Comment by caleblloyd** [comment]
LGTM

**Pull request merged**